presentation/match/src/main/java/com/giraffe/match/components/Carousel.kt

```diff
@@ -103,22 +103,27 @@ private fun CarouselItemContent(
     Box(
         modifier = modifier
             .clip(shape = RoundedCornerShape(Theme.radius.lg))
+            .background(Theme.color.background.card)
     ) {
         SafeIslamicImage(
             imageUrl = imageUrl,
             contentDescription = "image",
             contentScale = ContentScale.Crop,
             modifier = Modifier
                 .fillMaxSize()
-                .clip(shape = RoundedCornerShape(Theme.radius.lg))
+                .clip(shape = RoundedCornerShape(Theme.radius.lg)),
+            placeHolderTint = Theme.color.brand.secondary,
+            placeholderModifier = Modifier
+                .height(88.dp)
+                .width(64.dp)
```

Comment:
add background to placeHolder
```suggestion
            placeholderModifier = Modifier
                .background(Theme.color.background.card)
                .height(88.dp)
                .width(64.dp)
```